### PR TITLE
[GOVCMS-5205]: Gitlab CI integration for module audit generation

### PR DESCRIPTION
**Pull request by steveworley**
- Adds gitlab CI configuration to run scheduled jobs

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+
+image: govcms/govcms-ci:edge2
```
Comment (steveworley) [inline .gitlab-ci.yml:3]: https://github.com/govCMS/govcms-ci/pull/5

File: .ahoy.yml [hunk 2]
```diff
@@ -16,15 +16,15 @@ commands:
 
   cli:
     usage: Start a shell inside TEST container.
-    cmd: docker-compose exec test bash
+    cmd: docker-compose exec cli bash
```
Comment (steveworley) [inline .ahoy.yml:19]: I thought this was done a while ago - had to do this to stop some CI build errors in gitlab for some reason the connection between test > mysql was failing.

File: .gitlab-ci.yml (hunk 1 above)
Comment (stooit) [reply .gitlab-ci.yml:3]: This can change back to a stable release, has been merged

Comment (steveworley) [inline .gitlab-ci.yml:3]:
```suggestion
image: govcms/govcms-ci:latest
```

**Pull request merged**